src/models/pats.rs

```diff
@@ -85,8 +85,24 @@ bitflags::bitflags! {
         // perform analytics action
         const PERFORM_ANALYTICS = 1 << 30;
 
-        const ALL = 0b1111111111111111111111111111111;
-        const NOT_RESTRICTED = 0b00000011111111111111100111;
+        // create a collection
+        const COLLECTION_CREATE = 1 << 31;
+        // read a user's collections
+        const COLLECTION_READ = 1 << 32;
+        // write to a collection
+        const COLLECTION_WRITE = 1 << 33;
+        // delete a collection
+        const COLLECTION_DELETE = 1 << 34;
+        // read a collection's files
+        const COLLECTION_FILES_READ = 1 << 35;
+
+        // post an image
+        const IMAGE_POST = 1 << 36;
+        // delete an owned image
+        const IMAGE_DELETE = 1 << 37;
```

Comment:
these two are already associated with two other permissions. being able to write to threads or write to projects should be checked instead of 2 new permissions